opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservableEdition.tsx

```diff
@@ -0,0 +1,163 @@
+import useHelper from '../../../../utils/hooks/useHelper';
+import { useFormatter } from '../../../../components/i18n';
+import ThemeDark from '../../../../components/ThemeDark';
+import ThemeLight from '../../../../components/ThemeLight';
+import { QueryRenderer } from '../../../../relay/environment';
+import StixCyberObservableEditionContainer from './StixCyberObservableEditionContainer';
+import Loader, { LoaderVariant } from '../../../../components/Loader';
+import EditEntityControlledDial from '../../../../components/EditEntityControlledDial';
+import { StixCyberObservableEditionContainerQuery$data } from './__generated__/StixCyberObservableEditionContainerQuery.graphql';
+
+export const stixCyberObservableEditionQuery = graphql`
+  query StixCyberObservableEditionContainerQuery($id: String!) {
+    stixCyberObservable(id: $id) {
+      ...StixCyberObservableEditionContainer_stixCyberObservable
+      ...StixCyberObservable_stixCyberObservable
+    }
+  }
+`;
+
+interface StixCyberObservableEditionProps {
+  stixCyberObservableId: string,
+  open?: boolean,
+  handleClose?: () => void,
+}
+
+const StixCyberObservableEdition: FunctionComponent<StixCyberObservableEditionProps> = ({
+  stixCyberObservableId,
+  open: graphOpen,
+  handleClose: handleGraphClose,
+}) => {
+  const { t_i18n } = useFormatter();
+  const [open, setOpen] = useState<boolean>(false);
+  const [commit] = useApiMutation(stixCyberObservableEditionOverviewFocus);
+  const { isFeatureEnable } = useHelper();
+  const isFABReplaced = isFeatureEnable('FAB_REPLACEMENT');
+  const { palette: { mode } } = useTheme();
+  const theme = mode === 'dark'
+    ? ThemeDark()
+    : ThemeLight();
+
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => {
+    commit({
+      variables: {
+        id: stixCyberObservableId,
+        input: { focusOn: '' },
+      },
+    });
+    setOpen(false);
+  };
+  const transition = theme.transitions?.create
+    ? theme.transitions.create('width', {
+      easing: theme.transitions.easing?.sharp,
+      duration: theme.transitions.duration?.enteringScreen,
+    })
+    : undefined;
+
+  const renderClassic = () => (
+    <>
+      {isFABReplaced
+        ? (
+          <EditEntityControlledDial onOpen={handleOpen} onClose={() => {}}/>
+        ) : (
+          <Fab
+            onClick={handleOpen}
+            color="primary"
+            aria-label={t_i18n('Update')}
+            style={{
+              position: 'fixed',
+              bottom: 30,
+              right: 30,
+            }}
+          >
+            <Edit />
+          </Fab>
+        )
+      }
+      <Drawer
+        open={open}
+        anchor="right"
+        PaperProps={{
+          sx: {
+            zIndex: 1202,
+            minHeight: '100vh',
+            width: '50%',
+            position: 'fixed',
+            overflow: 'auto',
+            transition,
+            padding: 0,
+          },
+        }}
```

Comment:
I've tested again and it appears that doing so, the drawer gets a wrong z-index and appear below the main header bar.
If we want the drawer to appear on top of the whole screen, we need to apply zIndex on the Drawer and not the paper it seems.

```suggestion
        sx={{ 
            zIndex: 1202,
        }}
        PaperProps={{
          sx: {
            minHeight: '100vh',
            width: '50%',
            position: 'fixed',
            overflow: 'auto',
            transition,
            padding: 0,
          },
        }}
```